recipes/nand2tetris-core

```diff
@@ -0,0 +1,3 @@
+(nand2tetris-core :fetcher github
+                  :repo "CestDiego/nand2tetris.el"
+                  :files ("nand2tetris-core.el"))
```

Comment:
no, but how else do I  require the package? should I include core with nand2tetris...and then in company-nand2tetris I should require nand2tetris? (which requires core)